plugins/modules/swim_intent.py

```diff
@@ -809,6 +819,15 @@ def get_diff_import(self):
 
         try:
             import_type = self.want.get("import_type")
+
+            if not import_type:
+                self.msg = "For Importig SWIM Image, details are not provided"
```

Comment:
self.msg = "Error: Details required for importing SWIM image. Please provide the necessary information."

Reply:
added.